### PR TITLE
Expose the `keystore.path` in the YAML configuration

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: auditbeat/auditbeat.reference.yml [hunk 1]
```diff
@@ -1066,3 +1066,8 @@ logging.files:
 
 # Port on which the HTTP endpoint will bind. Default is 5066.
 #http.port: 5066
+#
```
Comment (ruflin) [inline auditbeat/auditbeat.reference.yml]: Nit: Lets remove the extra `#` here.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Rename beat.cpu.*.time metrics to beat.cpu.*.time.ms. {pull}6449[6449]
 - Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
 - Allow override of dynamic template `match_mapping_type` for fields with object_type. {pull}6691[6691]
+- Expose the `keystore.path` in the yaml configuration. {pull}6805[6805]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: How come this is  a breaking change? I would not even mention it in the changelog. Do we already have it in the docs?

Comment (ph) [reply CHANGELOG.asciidoc]: We don't have it in the docs, I can remove the changelog.

**Comment by ph** [comment]
@ruflin failling job on Jenkins should not be related to this PR.

**Pull request merged**